CHANGES

```diff
@@ -60,7 +60,8 @@ Tool Changes
   - automatically skip documentation of 'private' symbols
   - squashed generation of 'inline' in output
   - added support for --print-commands with chpldoc
-* added a quit() primitive to chpl-ipe for clean exits from the environment
+* chpl-ipe:
+  - added a quit() primitive for clean exits from the environment
```

Comment:
Ha!  I had it this way in an earlier draft, but decided that a 1-bullet list wasn't worthwhile, esp. given the lack of work on chpl-ipe this time around.  :)  Not worth changing back for my sake, though.